helm-chart/eoapi/initdb-data/settings/pgstac-settings.sql

```diff
@@ -0,0 +1,12 @@
+-- Apply pgstac settings
```

Comment:
@emmanuelmathot is there a reference for what this is doing exactly / what settings these are, etc? If these are settings, do we need to make them configurable in `values`? Am guessing this is something just a bit silly to get things to work, but might be nice to have a comment with a link to what these settings are or so?

Reply:
Yes, the idea is to have them in the values. I actually wanted to have a proper mechanism via pypgstac cli with something like pypgstac load settings that may be used not only in this chart.